testlib/src/test/java/com/diffplug/spotless/generic/LicenseHeaderStepTest.java

```diff
@@ -16,19 +16,27 @@
 package com.diffplug.spotless.generic;
 
 import java.io.File;
+import java.io.IOException;
 import java.nio.charset.StandardCharsets;
+import java.time.YearMonth;
 
 import org.junit.Assert;
 import org.junit.Test;
 
 import com.diffplug.spotless.FormatterStep;
 import com.diffplug.spotless.ResourceHarness;
 import com.diffplug.spotless.SerializableEqualityTester;
+import com.diffplug.spotless.StepHarness;
 
 public class LicenseHeaderStepTest extends ResourceHarness {
 	private static final String KEY_LICENSE = "license/TestLicense";
 	private static final String KEY_FILE_NOTAPPLIED = "license/MissingLicense.test";
 	private static final String KEY_FILE_APPLIED = "license/HasLicense.test";
+	private static final String KEY_LICENSE_WITH_YEAR = "license/TestLicencseWithYear";
+	private static final String KEY_FILE_WITHOUT_LICENSE = "license/MissLicenseWithYear.test";
+	private static final String KEY_FILE_WITH_PREVIOUS_YEAR = "license/LicenseWithPreviousYear.test";
+	private static final String KEY_FILE_WITH_PREVIOUS_YEARS = "license/LicenseWithPreviousYears.test";
+	private static final String KEY_FILE_WITH_CURRENT_YEAR = "license/LicenseWithYear.test";
```

Comment:
I believe that `KEY_FILE_WITH_PREVIOUS_YEAR` and `KEY_FILE_WITH_PREVIOUS_YEARS` aren't used. If they are indeed unused, and if there are any other unused constants here that I've missed, then I think they should be removed.

Reply:
good catch